collated/main/tpq95.md

```diff
@@ -0,0 +1,585 @@
+                                } catch (IllegalValueException e) {
+                                    e.printStackTrace();
+                                }
+                                String finalTaskDate = taskDate;
+                                try {
+                                    //ensure that Deadline/Startdate is valid, after computer is invented
+                                    assert LocalDate.parse(finalTaskDate, formatter).getYear()
+                                            >= (LocalDate.now().getYear() - 100);
+                                    if (item.equals(LocalDate.parse(finalTaskDate, formatter))) {
+                                        if ((bCount == 0) && (dCount == 0)) {
+                                            dCount++;
+                                            s.append(dCount + " Deadline");
+                                        } else if ((bCount > 0) && (dCount == 0)) {
+                                            dCount++;
+                                            s.append(" + " + dCount + " Deadline");
+                                        } else if (dCount > 0) {
+                                            dCount++;
+                                            int endIndex = s.indexOf(" Deadline");
+                                            s.replace(endIndex - 1, endIndex, dCount + "");
+                                            if (dCount == 2) {
+                                                s.append("s");
+                                            }
+                                        }
+
+                                        if (bCount == 0) {
+                                            colour = new StringBuilder("-fx-background-color: #ff444d;");
+                                        } else {
+                                            colour = new StringBuilder("-fx-background-color: #feff31;");
+                                        }
+                                    }
+                                } catch (DateTimeParseException exc) {
+                                    logger.warning("Not parsable: " + task.getDeadline().date);
+                                    throw exc;
+                                }
+                            }
+                        }
+                        if (s.length() != 0) {
+                            setTooltip(new Tooltip(s.toString()));
+                        }
+                        setStyle(colour.toString());
+                    }
+                };
+            }
+        };
+        return dayCellFactory;
+    }
+
+    /**
+     * Select the date on calendar
+     * @param date
+     */
+    /*private void selectDate(DatePickerContent content, String date) {
+        ObservableList<Node> dateCellList = content.getChildren();
+        LocalDate localDate = LocalDate.parse(date, formatter);
+
+        logger.info("day of month: " + localDate.getDayOfMonth());
```

Comment:
This should probably be collated under unused, or remove it if its already been superseded.